package.json

```diff
@@ -22,8 +22,8 @@
     "test:browser": "npx jest --runInBand --verbose --config test/browser/jest.config.js --forceExit",
     "test:browser:ci": "npm run install:puppeteer && npm run test:browser",
     "install:puppeteer": "npm install --save-dev jest-puppeteer puppeteer @types/puppeteer",
-    "prepare": "node ./scripts/tsc.js && node ./scripts/build.js",
-    "install": "node ./scripts/install.js --production"
+    "prepare": "node ./scripts/tsc.js",
+    "install": "node ./scripts/install.js"
```

Comment:
I can't figure out why that `--production` flag was ever there ???

I see it was added in [PR #329](https://github.com/awslabs/aws-crt-nodejs/pull/329/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R23)

but why?

Reply:
No idea, but it is not doing anything and makes me confused. So, removed it

Reply:
The `install.js` and `build.js` is all our code, and none of those uses `--production` anywhere.

Reply:
its a process wide argv, so something outiside of our code might have been picking it, but i doubt it. looks unused